### PR TITLE
Add ATAC

### PR DESCRIPTION
**Pull request by Julien-cpsn**
ATAC is Arguably a Terminal API Client. Feature-full, free, open-source, offline and account-less.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Aside from those listed here, many other apps and libraries can be easily be fou
 - [TaskUI](https://github.com/thmshmm/taskui) - Simple Terminal UI for Task / taskfile.dev.
 - [tenere](https://github.com/pythops/tenere) - TUI interface for LLMs written in Rust.
 - [Yazi](https://github.com/sxyazi/yazi) - Blazing fast terminal file manager written in Rust, based on async I/O.
+- [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client made with the lovely ratatui. ATAC is free, open-source, offline and account-less.
```
Comment (orhun) [inline README.md:86]: Let's simplify the description a bit 🐻 

```suggestion
- [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client for your terminal.
```

Comment (Julien-cpsn) [reply README.md:86]: Thank you!

Comment (orhun) [inline README.md:86]:
```suggestion
- [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client for your terminal.
```

Comment (orhun) [reply README.md:86]: Can you accept my suggestion? 👀

**Review by orhun — APPROVED** [review]

**Pull request merged**